coding-concepts.js

```diff
@@ -5,39 +5,61 @@
 // First, predict the outcome based on your understanding of the code.
 // Then, uncomment the console.log() and verify the output. Briefly explain why your initial answer was correct or incorrect.
 
+
+
 // --------------------1) What will this log?
 
 const cohort = "Golf 2022"
-// console.log(cohort.split(""))
+//console.log(cohort.split(""))
 
-// a) Your answer:
+// a) Your answer: ["G","o","l","f","2","0","2","2"]
 // b) Verify and explain:
+  // The string method .split() will split all the characters in the string into individual elements and put those elements into an array. However, I was incorrect in that the new array returns single quotes AND includes the space between Golf and 2022. The space is an element as it has an actual index so it is included into the array.
+
+
 
 // --------------------2) What will this log?
 
 const greeter = (name) => {
   `Hello, ${name}!`
 }
-// console.log(greeter("LEARN Student"))
+//console.log(greeter("LEARN Student"))
 
-// a) Your answer:
+// a) Your answer: 'Hello, LEARN Student'
 // b) Verify and explain:
+// I was wrong. This is more a matter of viewing the code carefully which I obviously did not. Whenever a function returns an output of undefined, the first thing you should do is to check and see whether there is a missing return statement. I can't believe I didn't look for a return statement before I ran the console log. You got me!
```

Comment:
This can be one of the harder parts of being a developer, is being really attentive to small details.  I have seen a group of 5 people stumped on an error for an hour because of a simple typo.